components/templates/src/builtins/atom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <feed xmlns="http://www.w3.org/2005/Atom" xml:lang="{{ lang }}">
 	<title>{{ config.title }}
-		{%- if term %} - {{ term.name }}
+		{%- if term %} - {{ term.name | replace(from="-", to=" ") |  title}}
```

Comment:
sadly that doesn't work if you have unicode :/
The deduplication should happen without needing any changes to the templates

Reply:
okay I'll have a look and see what I can do